yarn-project/circuit-types/src/messaging/l1_actor.ts

```diff
@@ -0,0 +1,43 @@
+import { randomInt } from '@aztec/foundation/crypto';
+import { EthAddress } from '@aztec/foundation/eth-address';
+import { Fr } from '@aztec/foundation/fields';
+import { BufferReader, serializeToBuffer } from '@aztec/foundation/serialize';
+
+/**
+ * The sender of an L1 to L2 message.
+ */
+export class L1Actor {
+  constructor(
+    /**
+     * The sender of the message.
+     */
+    public readonly sender: EthAddress,
+    /**
+     * The chain id on which the message was sent.
```

Comment:
Not only sent from, could also be the one that it is sent to, so the recipient.

Reply:
Addressed